app/views/events/_show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if check %>
+  <h3>Title: <%= @event.title %></h3>
+  <p><strong>Event date: </strong><%= @event.event_date %></p>
+  <p><strong>Description: </strong><%= @event.description %></p>
+
+  <% if @event.attendees.empty? %>
+    <p>No attandees</p>
+  <% else %>
+    <strong>Attendees:</strong>
+    <ul>
+      <% @event.attendees.each do |attendee| %>
+        <li><%= link_to attendee.full_name, user_path(attendee.id) %></li>
+      <% end %>
+    </ul>
+  <% end %>
+
```

Comment:
- [ ] There should be no logic in the views. Please encapsulate all of the view-related logic in your view helpers: **(app/helpers/*_helper.rb)**